src/components/RequestGroupComposition/Request.vue

```diff
@@ -177,13 +200,13 @@
               header="Generated Mosaic"
               :show-accept="expansion.expanded.length > 0"
               @close="cancelExpansion"
-              @submit="acceptExpansionForKeyOnObject(request, 'configurations')"
+              @submit="acceptMosaic()"
             >
               <data-loader
                 :data-loaded="expansion.status.loaded"
                 :data-not-found="expansion.status.notFound"
                 :data-load-error="expansion.status.error"
-                not-found-message="There are no pointing in the generated mosaic. Please update your mosaic parameters and try again."
+                not-found-message="There are no pointings in the generated mosaic. Please update your mosaic parameters and try again."
```

Comment:
Is it possible to expand a mosaic and end up with no pointings? I feel like this data loading error is more indicative of a connection issue between the frontend and backend or an issue with the backend API than it is with there being no pointings...

Reply:
Hmm yeah. So I just double checked and technically yes you can end up with zero pointings if you input something like `num_points=0`. This is probably something that we should validate for in the backend. In any case, this message should only be displayed if the response returns successfully, but there are no configurations in the response. If there is a connection error, a 500 error, a validation error, or something like that, then a different error message is displayed - [see here](https://github.com/observatorycontrolsystem/ocs-component-lib/blob/main/src/components/RequestGroupComposition/Request.vue#L189). If we do update our backend validation to catch all the cases where we end up returning zero configurations, then yes this message should never be displayed, but I believe it accurately captures that there are not configurations.

Reply:
ah got it, we should validate against zero points or rows/columns in the serializer